test/style_trans_23_to_30.js

```diff
@@ -867,6 +867,58 @@ describe('cartocss transformation from 2.3.x to 3.0.x', function() {
         }]
     };
 
+    var urlSuite = {
+        symbolizer: 'line-pattern',
+        testCases: [{
+            description: 'should handle url enclosed by simple quotes',
+            input: [
+                '#layer {',
+                '  line-width: 5;',
```

Comment:
About inline comments, I think they are covered [here](https://github.com/CartoDB/grainstore/blob/master/test/style_trans_23_to_30.js#L1048-L1110)

Or you meant a test with comments and url value. ??